crates/script/src/lib.rs

```diff
@@ -202,6 +203,10 @@ pub struct ScriptArgs {
     #[arg(long, env = "ETH_TIMEOUT")]
     pub timeout: Option<u64>,
 
+    /// Only shows the transactions that would be sent without actually broadcasting them.
```

Comment:
This is incorrect, it would list the transactions alongside broadcasting them if `--broadcast` is passed